datagateway_api/src/resources/search_api_endpoints.py

```diff
@@ -8,77 +11,91 @@
     get_with_id,
 )
 
+log = logging.getLogger()
+
 
-def get_search_endpoint(name):
+def get_search_endpoint(endpoint_name, entity_name):
     """
     TODO - Add docstring
     """
 
     class Endpoint(Resource):
         def get(self):
-            return get_search(name), 200
+            filters = get_filters_from_query_string("search_api", entity_name)
+            log.debug("Filters: %s", filters)
+            return get_search(endpoint_name, entity_name, filters), 200
 
         # TODO - Add `get.__doc__`
 
-    Endpoint.__name__ = name
+    Endpoint.__name__ = entity_name
     return Endpoint
 
 
-def get_single_endpoint(name):
+def get_single_endpoint(endpoint_name, entity_name):
     """
     TODO - Add docstring
     """
 
     class EndpointWithID(Resource):
         def get(self, pid):
-            return get_with_id(name, pid), 200
+            filters = get_filters_from_query_string("search_api", entity_name)
```

Comment:
Do we not need to pass `filters` to the `get_with_id` method? Same question applies for the other endpoint methods below.

Reply:
This is fixed in #302, the code before the implementation of the endpoints was basically a skeleton to prompt me when I got to implementing them.